### DISCUSSION TITLE
Please add Irish to Home Assistant - Frontend

### DISCUSSION THREAD

**Discussion by aindriu80** [home-assistant/frontend#19797] [Localization]
Hi,

Can you please add Irish - Gaeilge to Home Assistant - Frontend:- https://app.lokalise.com/projects?

English Name: Irish
Native Name: Gaeilge
Language Code: ga

Go raibh míle maith agat

**Comment by aindriu80** [comment]
Can someone please add Irish as per https://developers.home-assistant.io/docs/translations guide

**Comment by niallmca** [comment]
By daid, tá sí ann anois.
Nach iontach go deo é seo :)
Ard-fhear, a Aindriú
